python/lsst/ts/externalscripts/auxtel/latiss_acquire.py

```diff
@@ -0,0 +1,604 @@
+                description: Perform re-acquisition of target without initial slew. Assumes target
+                    is near final position.
+                type: boolean
+                default: False
+
+            additionalProperties: false
+            if:
+              properties:
+                do_reacquire:
+                  const: True
+              required: []
+            elif:
+              properties:
+                object_ra:
+                  const: null
+                object_dec:
+                  const: null
+              required: ["object_name", "program", "reason"]
+            else:
+              required: ["object_name", "object_ra", "object_dec", "program", "reason"]
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+
+        # Instantiate BestEffortIsr -
+        self.best_effort_isr = self.get_best_effort_isr()
+
+        # instantiate the quick measurement class
+        qm_config = QuickFrameMeasurementTask.ConfigClass()
+        self.qm = QuickFrameMeasurementTask(config=qm_config)
+
+        self.do_reacquire = config.do_reacquire
+
+        if not self.do_reacquire:
+            self.object_name = config.object_name
+
+        self.object_ra = config.object_ra
+        self.object_dec = config.object_dec
+
+        # config for the single image acquisition
+        self.acq_grating = config.acq_grating
+        self.acq_filter = config.acq_filter
+        self.acq_exposure_time = config.acq_exposure_time
+
+        # Max number of iterations to perform when putting source in place
+        self.max_acq_iter = config.max_acq_iter
+
+        # Tolerance in arcsec for distance from main source centroid to
+        # relevant sweet spot
+        self.target_pointing_tolerance = config.target_pointing_tolerance
+        self.target_pointing_verification = config.target_pointing_verification
+
+        self.reason = config.reason
+
+        self.program = config.program
+
+        self.time_on_target = config.time_on_target
+
+        if self.time_on_target is None:
+            self.time_on_target = 0.0
+            self.azimuth_wrap_strategy = WrapStrategy.MAXTIMEONTARGET
+        else:
+            self.azimuth_wrap_strategy = WrapStrategy.OPTIMIZE
+
+        self.rot_type = config.rot_type
+        self.rot_value = config.rot_value
+
+        self.do_manual_final_position = config.do_manual_final_position
+
+        if self.do_manual_final_position:
+            self.target_position = self.do_manual_final_position
+        else:
+            self.target_position = latiss_constants.sweet_spots[self.acq_grating]
+
+        self.image_in_oods_timeout = 15.0
+
+    def set_metadata(self, metadata):
+        slew_duration = 0 if self.do_reacquire else 60
```

Comment:
You could add a `estimated_slew_time` to the script configuration. The scheduler can pass this information and you could use it here instead of a hard coded value of 60.